### ISSUE TITLE
kubernetesonarm/loadbalancer in in DockerHub

### ISSUE THREAD

**Issue by saturnism** [luxas/kubernetes-on-arm#50]
```
Failed	Failed to pull image "kubernetesonarm/loadbalancer": Error: image kubernetesonarm/loadbalancer:latest not found
```

**Comment by luxas** [comment]
Have you tested to build `loadbalancer` by hand? If not, try.
I haven't got it working yet, so therefore it isn't pushed.

**Comment by larmog** [comment]
luxas: 

I'm using the loadbalancer. I built the binary from k8s contrib and changed the template.cfg (I found some errors in the https config from the master branch) and used your Dockerfile to build a new image.
I't works for me, haven't had any problems with serviceaccount.
/Regards

**Comment by luxas** [comment]
@larmog Awesome! Could you please share the details how you changed `template.cfg`? It was probably because of that I didn't get it working. Thanks for getting in touch

**Comment by larmog** [comment]
I've issued a pull request, take a look and see what you think. :)

**Comment by luxas** [comment]
Fixed with `v0.6.5`
